leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -93,7 +107,7 @@ public RESPONSE call() throws Exception {
             while (acksRecieved < quorumSize) {
                 try {
                     // check if quorum is impossible (nack quorum failure)
-                    if (nacksRecieved > remotes.size() - quorumSize) {
+                    if (shortcircuitIfQuorumImpossible && nacksRecieved > remotes.size() - quorumSize) {
```

Comment:
It's the point of the night where my brain is starting to fail - but am I right in reading that you're skipping this check of achieving quorum for the PaxosSyncronizer unit tests? Is it not possible for us to wait until node 1 has finished learning the value?